### DISCUSSION TITLE
Does this support serializing the engine?

### DISCUSSION THREAD

**Discussion by dustinlacewell** [sebastienros/jint#998] [Q&A] [answered]

**Comment by lahma** [comment, marked as answer]
No. You can use for example interop to inject state object that you can then use to serialize/maintain state between restarts.

**Reply by dustinlacewell** [reply]
I have been looking around for a runtime that will allow me to pause something using async/await and restart it without forcing the program design into some sort of command pattern with replay, etc. It's a legacy thing... Anyway thank you for your answer.
